task/git-clone/0.9/git-clone.yaml

```diff
@@ -107,7 +107,7 @@ spec:
       description: |
         Absolute path to the user's home directory.
       type: string
-      default: "/home/git"
+      default: "/workspace/nonroot"
```

Comment:
this can probably change user experience for all the current users. Can you please bump the version of git-clone? Also can you please help with the Pipelines version and the platform on which you are facing this issue? Probably it will be worth adding in the readme of the new version of git-clone task `v0.10`.
@khrm @natifridman do let me know your thoughts around this 😄

Reply:
Created new version `v0.10`
@vinamra28 Let me know if I missed something. Where would you add the Pipelines version?